### PR TITLE
BAQE-2275 - kie-benchmarks repository reorganization (7x branch)

### PR DESCRIPTION
**Pull request by hellowdan**
-removed optaplanner-8-benchmarks module
-optaplanner-7-benchmarks module renamed to optaplanner-benchmarks

### CODE REVIEW

**Review by dupliaka — APPROVED** [review]
Looks good to me, please make it non draft so we can see the GA results

**Comment by hellowdan** [comment]
> Looks good to me, please make it non draft so we can see the GA results

Thanks @dupliaka . It's ready for review now :)

**Comment by dupliaka** [comment]
I see the pull request checks are set up for push and pull request events for 7.x.x (7.56.x as example) 
So in case you want to enable it here you need to add -
```
on:
  push:
    branches:
      - main
      - 7.x
```

**Comment by hellowdan** [comment]
> I see the pull request checks are set up for push and pull request events for 7.x.x (7.56.x as example) So in case you want to enable it here you need to add -
> 
> ```
> on:
>   push:
>     branches:
>       - main
>       - 7.x
> ```

Thanks @dupliaka!

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -4,16 +4,14 @@ on:
   push:
     branches:
       - main
```
Comment (MarianMacik) [inline .github/workflows/pull_request.yml:6]: Based on the docs [1], the workflow file is taken from the target branch, so we can even exclude main and leave just 7.x and 7.x.x branches. For main we could set it up similarly - main and every forked branch from it (8.x.x). Wdyt?

Comment (dupliaka) [reply .github/workflows/pull_request.yml:6]: I do not see [1] but looks logical to keep the branch checks specification within the concrete branch

**Review by MichalCervenansky — APPROVED** [review]
Looks good, thanks Daniel!

File: .github/workflows/pull_request.yml (hunk 1 above)
Comment (hellowdan) [reply .github/workflows/pull_request.yml]: Thanks @MarianMacik @dupliaka ! I've just sent another commit which removes the main branch from the check. Will do the same on main branch.

**Review by MarianMacik — APPROVED** [review]

**Review by mariofusco — APPROVED** [review]

**Pull request merged**